### PR TITLE
Hit 100% with branch coverage

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: tests/contrib/test_django_orm.py [hunk 1]
```diff
@@ -130,6 +133,9 @@ def test_field_value_to_string_none(self):
         value_str = self.fake_model_field.value_to_string(self.fake_model)
         self.assertEqual(value_str, None)
 
+    def test_flow_with_null(self):
+        flow = FlowField(null=True)
```
Comment (dhermes) [inline tests/contrib/test_django_orm.py]: Can we test something here?

File: tests/contrib/test_django_orm.py [hunk 2]
```diff
@@ -95,6 +95,9 @@ def test_field_value_to_string_none(self):
         value_str = self.fake_model_field.value_to_string(self.fake_model)
         self.assertEqual(value_str, None)
 
+    def test_credentials_with_null(self):
+        credentials = CredentialsField(null=True)
```
Comment (dhermes) [inline tests/contrib/test_django_orm.py]: Ditto, test something?

**Comment by dhermes** [comment]
@waprin the misses are in `tests/contrib/test_django_orm.py`. Looks like you have some lines that don't get executed?

File: tests/contrib/test_django_orm.py [hunk 3]
```diff
@@ -246,5 +283,18 @@ def save(self):
...
 
+class FakeCredentialsModelMockNoSet(object):
+
+    def __init__(self, set_store=False, *args, **kwargs):
+        self.model = FakeCredentialsModelMock
+        self.saved = False
+        self.deleted = False
+
+    def save(self):
+        self.saved = True
```
Comment (dhermes) [inline tests/contrib/test_django_orm.py]: ISTM this is the candidate for a line that doesn't get executed.

**Comment by waprin** [comment]
Ok, flipped the logic so those tests have some asserts, now at ONE HUNDRED PERCENT.

**Comment by dhermes** [comment]
:100: :100: :100:

**Comment by dhermes** [comment]
:sunglasses:

**Comment by dhermes** [comment]
Merge merge merge merge merge!

**Pull request merged**